routes/projects/files/get-one.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * GET /projects/:username/:projectName/files/:pathToFile
+ * The :pathToFile parameter must be URI encoded
+ * Returns the contents of the given file in the form:
+ * {
+ *   type: <string> content-type,
+ *   path: <string>,
+ *   size: <integer>,
+ *   link: <string>
```

Comment:
yes that is correct. i think serving a blob from the API is not a great idea. passing a representation of the file and not the file itself seems best in my opinion. the blob should be served statically.